### PR TITLE
feat(client-go): add perms endpoint

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

**Comment by mboersma** [comment]
Also needs rebasing. :circus_tent:

**Comment by Joshua-Anderson** [comment]
Rebased!

**Comment by technosophos** [comment]
Code LGTM

File: client-go/parser/perms.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	if err != nil {
+		return err
+	}
+
+	admin := args["--admin"].(bool)
+
+	return cmd.PermsList(safeGetValue(args, "--app"), admin)
+}
+
+func permCreate(argv []string) error {
```
Comment (bacongobbler) [inline client-go/parser/perms.go]: the function names seem inconsistent here. To match with the call, let's make this `permsCreate()`

File: client-go/parser/perms.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	}
+
+	app := safeGetValue(args, "--app")
+	username := args["<username>"].(string)
+	admin := args["--admin"].(bool)
+
+	return cmd.PermCreate(app, username, admin)
+}
+
+func permDelete(argv []string) error {
```
Comment (bacongobbler) [inline client-go/parser/perms.go]: same here; `permsDelete()`

**Comment by bacongobbler** [comment]
small nitpicks, otherwise LGTM

File: client-go/parser/perms.go (hunk 1 above)
Comment (Joshua-Anderson) [inline client-go/parser/perms.go]: For my code I went with singular versions when dealing with a single object (creating, removing, getting) and plural with commands that deal with multiple objects (listing). I'm happy to change if it is confusing, but is probably a client-go wide change.

Comment (bacongobbler) [inline client-go/parser/perms.go]: nah as long as we're consistent

**Pull request merged**